apis/topology/v1alpha1/common.go

```diff
@@ -90,6 +91,12 @@ type TopologyCommonSpec struct {
 	// +kubebuilder:validation:Enum=disabled;critical;warn;info;debug
 	// +optional
 	LauncherLogLevel string `json:"launcherLogLevel,omitempty"`
+	// Resources is a mapping of nodeName (or "default") to kubernetes resource requirements -- any
```

Comment:
what does `default` key mean? Is it a requirement that is applied to all topology nodes?

Reply:
yeah exactly, will add some more clarity to that doc